cli/src/pcluster/utils.py

```diff
@@ -1076,7 +1076,28 @@ def read_remote_file(url):
         if urlparse(url).scheme == "s3":
             match = re.match(r"s3://(.*?)/(.*)", url)
             bucket, key = match.group(1), match.group(2)
-            file_contents = boto3.resource("s3").Object(bucket, key).get()["Body"].read().decode("utf-8")
+            try:
+                file_contents = (
+                    boto3.resource("s3", region_name="us-east-1")
```

Comment:
We should move this call in the `pcluster/aws/s3_resource.py` module, in the `S3Resource` class.